packages/muon/components/inputter/src/inputter-styles.slotted.css

```
@@ -194,7 +194,7 @@ light-dom {
     @extend %input-textarea;
   }
 
-  &[mask] {
+  &[mask]:not([mask='']) {
```

I believe double quotes should be used:

```suggestion
  &[mask]:not([mask=""]) {
```